### PR TITLE
techdept(crux-ui): add e2e environment setup

### PR DESCRIPTION
**Pull request by m8vago**
End to end test environment setup with:
 - an existing user
 -  a verified e-mail
 - already logged in
 - already has a team

### CODE REVIEW

File: web/crux-ui/e2e/utils/common.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  const url = replacePort(baseURL, '4437')
+  return new MailSlurper(url)
+}
+
+export const mailslurpFromConfig = (config: FullConfig) => {
+  const { baseURL } = config.projects[0].use
+  return mailslurpFromBaseURL(baseURL)
+}
+
+export const cruxAddressOfConfig = (config: FullConfig) => {
```
Comment (robot9706) [inline web/crux-ui/e2e/utils/common.ts:29]: Could be called `cruxAddressFromConfig`.

File: web/crux-ui/e2e/utils/global-setup.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  await page.waitForURL(ROUTE_LOGIN)
+  await page.locator('input[name=email]').fill(USER_EMAIL)
+  await page.locator('input[name=password]').fill(USER_PASSWORD)
+  await page.locator('button[type=submit]').click()
+
+  await page.waitForURL(ROUTE_TEAMS_CREATE)
+  await page.locator('input[name=name]').fill(USER_TEAM)
+  await page.locator('button[type=submit]').click()
+
+  await page.screenshot({ path: screenshotPath('cica'), fullPage: true })
```
Comment (robot9706) [inline web/crux-ui/e2e/utils/global-setup.ts:48]: 🐱? :D

**Review by robot9706 — CHANGES_REQUESTED** [review]

File: web/crux-ui/e2e/utils/global-setup.ts (hunk 2 above)
Comment (polaroi8d) [reply web/crux-ui/e2e/utils/global-setup.ts:48]: Where is my screenshots?

Comment (polaroi8d) [reply web/crux-ui/e2e/utils/global-setup.ts:48]: I dont see this folder in the PR

Comment (m8vago) [reply web/crux-ui/e2e/utils/global-setup.ts:48]: cause it was already created in a previous pr

Comment (polaroi8d) [reply web/crux-ui/e2e/utils/global-setup.ts:48]: What about if we create artifacts for PRs?

Comment (m8vago) [reply web/crux-ui/e2e/utils/global-setup.ts:48]: i see the problem now. just move common.ts and that messed up some stuff.

**Review by robot9706 — APPROVED** [review]
🐱

**Review by robot9706 — APPROVED** [review]
LGTM 😎

File: web/crux-ui/docker-compose.yaml [hunk 3]
```diff
@@ -8,13 +8,13 @@ services:
...
       - KRATOS_URL=$KRATOS_URL
-      - CRUX_ADDRESS=$CRUX_ADDRESS
+      - CRUX_API_ADDRESS=$CRUX_API_ADDRESS
       - RECAPTCHA_SECRET_KEY=$RECAPTCHA_SECRET_KEY
       - RECAPTCHA_SITE_KEY=$RECAPTCHA_SITE_KEY
     restart: unless-stopped
     volumes:
       - crux-ui-certs:/app/certs
-    networks:
+    networks:
```
Comment (c3ppc3pp) [inline web/crux-ui/docker-compose.yaml:17]: extra space

File: .github/workflows/web-crux-ui.yml [hunk 4]
```diff
@@ -14,21 +14,14 @@ env:
   # Environments
   REGISTRY: ghcr.io
   IMAGE_NAME: dyrector-io/dyrectorio/web/crux-ui
-  WORKING_DIRECTORY: web/crux-ui
```
Comment (c3ppc3pp) [inline .github/workflows/web-crux-ui.yml:17]: You were a bit too eager with deletions

**Review by c3ppc3pp — APPROVED** [review]
Fix the pipeline, otherwise LGTM

**Pull request merged**